client/src/app/project-registration/contact-field-group/contact-field-group.component.ts

```diff
@@ -54,15 +53,16 @@ export class ContactFieldGroupComponent implements OnInit {
     // default
     correspondingCtrl.setValue(true);
 
-    this.aai.getUserSubject().pipe(first()).subscribe(user => {
-      this.userInfo = user ? user.profile : null;
+    const user = this.aai.getUser().value;
```

Comment:
@MightyAx 
Is this a safe change to make? If the `this.aai.getUser()` subjects has no values at the time of initialisation then I assume `const user` will be null/undefined and then below code will never be called, or throw an error.

Reply:
You can't get to the contact-field-group without logging in.

Reply:
The call to `AaiService.loggedIn(user)` checks If the user is null

Reply:
ah, do you mean the `.value` property?
BehaviourSubjects have an initial value supplied in the constructor
that initial value or whatever the latest value provided by the `.next` call is provided when you call `.value` or .`GetValue()`
```
private cachedUser$: BehaviorSubject<User> = new BehaviorSubject<User>(null);
this.cachedUser$.value //initially null
this.cachedUser$.next(<User>Bob) //will trigger call to any subscribers
this.cachedUser$.value // now Bob
this.cachedUser$.getValue() // Also Bob
```

Reply:
The below code doesn't run in that case. Is that the behaviour we want, or do we want the below code to run at some point, for example when the user subject does have a value?

Reply:
Yeah, but if the user hasn't logged in we can't get their name and email from their user account, so it's correct that we don't run the code in the case that they haven't logged in.

Reply:
That makes sense, but can't we just subscribe to the user subject so that we run the below code when a non-null user becomes available?


e.g

```
this.aai.getUser().pipe(
   filter(maybeUser => mayberUser !== null),
   take(1)
).subscribe(user => {
  // below code
})
```

The prior code is similar but